### ISSUE TITLE
"Getting started" and "installation" are in conflict with each other

### ISSUE THREAD

**Issue by maestromac** [forem/forem-docs#11]
I'm having trouble understanding the purpose of the "getting started" section. My understanding is maybe that's the steps on how to start and use the app but it's currently repeating the installation steps and missing some crucial ones. My concern is that new contributors might be confused about which of the two to start off with.

So my take right now maybe removes `Forking the Repository`, `Configuring Environment Variables`, and `Preparing the Database` section because those are already covered by the `installation` section.

Would be interested to hear your take @cmgorton

**Comment by cmgorton** [comment]
I agree with this Mac. The only thing I can't find though maybe just overlooking is the `Preparing the Database` part in the Installation section. I see all the other parts you mentioned though and agree it is repetitive to have them in both sections. 

Maybe we should just rename the Installation section as `Getting Started`

**Comment by Zhao-Andy** [comment]
I like renaming the installation section as "Getting Started" -- we can also move specific OS installation guides under a subfolder there, too.

Some of the pages in "Getting Started" should probably be moved too, like "creating a branch", "suggested workflow", "preparing a PR" should maybe go in the "Contributing Guide".

**Comment by maestromac** [comment]
@cmgorton `Preparing the Database` is kind of all covered in `bin/setup` step. 

Alright let me move some stuff around
